proto/cosmos/accesscontrol/genesis.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+package cosmos.accesscontrol.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/accesscontrol/constants.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/accesscontrol/types";
+
+message MessageDependencyMapping {
+    MessageType message_type = 1;
+    Module module_name = 2;
+    repeated string resource_dependencies = 3;
```

Comment:
I just realized this should probably be a list of AccessOps instead of just a list of strings 


```
type AccessOp struct {
    MsgKey string
    Type AccessType
    Resource ResourceType
    IdentifierTemplate string // can be generalized
}
```

Reply:
```
message AccessOp {
    Type AccessType = 1;
    Resource ResourceType = 2;
    IdentifierTemplate string = 3;
}

message MessageDependencyMapping {
    MessageType message_type = 1;
    Module module_name = 2;
    repeated AccessOp ordered_access_op = 3;
}
```